examples/snowpack/src/pages/tutorials/getting-started.md

```diff
@@ -80,7 +80,7 @@ Add the Snowpack development server to `package.json` under as the `start` scrip
 Run the following on the command line to start the Snowpack development server
 
 ```
-npm start
```

Comment:
you might need to manually review the content of the snowpack example since it seems to reference these commands in it

Reply:
yup, I'd revert any changes to `examples/snowpack`, this example was our original "kitchen sink" and should be removed any day now.